storage/innobase/xtrabackup/src/read_filt.cc

```diff
@@ -28,6 +28,9 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
 #include "fil_cur.h"
 #include "xtrabackup.h"
 
+/** map of tablespace_id, that experienced an inplace DDL during a backup op */
+extern std::map<space_id_t, bool> index_load_map;
+
```

Comment:
Same here, this is confusing and we are following no logic. We should be externing the variable on the header file where it's been declared, not where you will use it. If you use it in two external places, which logic to follow in regards to where to extern it?

You declared this on `log0recv.cc` so the logical place for this extern is on `include/log0recv.h`.

Take for example `index_load_list`:
 - Declared https://github.com/percona/percona-xtrabackup/blob/8.0/storage/innobase/log/log0recv.cc#L107
 - Extern on it's corresponding header file - https://github.com/percona/percona-xtrabackup/blob/8.0/storage/innobase/include/log0recv.h#L56